src/plone/staticresources/static/components/svg-country-flags/README.md

```diff
@@ -0,0 +1,50 @@
+country-flags
+=============
+
+You can view all the flags here: http://hjnilsson.github.io/country-flags/
```

Comment:
This website does not exist. It's prpbably this one: https://hampusborgos.github.io/country-flags/
Please change.

Reply:
probably this was copied from the original npm pkg. It was also fixed upstream already https://github.com/hampusborgos/country-flags/commit/2c5cc97ffdb444a830efb3da6f9ef4149d1220c5#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

Reply:
@thet thanks for the hints. I will take it into account.

Reply:
Ah this comes from the external package. Didn't see that.